### PR TITLE
Upgrade deps etc.

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,32 +10,32 @@
...
     "shelljs": "0.7.7",
     "gh-pages": "0.10.0",
 
-    "kw-web-suite": "2.2.1",
-    "kw-doc": "1.0.19",
+    "kw-web-suite": "6.0.2",
+    "kw-doc": "1.1.4",
 
-    "babel-polyfill": "6.23.0"
+    "babel-polyfill": "6.26.0"
```
Comment (jourdain) [inline package.json]: I think you can remove it due to babel-preset-env

File: webpack.config.js [hunk 2]
```diff
@@ -34,24 +42,48 @@ var loaders = [
     loader: 'html-loader',
   }, {
     test: /\.glsl$/,
-    loader: 'shader',
+    loader: 'shader-loader',
   }, {
     test: /\.js$/,
     include: /node_modules(\/|\\)paraviewweb(\/|\\)/,
-    loader: 'babel?presets[]=es2015,presets[]=react',
+    loader: 'babel-loader?presets[]=es2015,presets[]=react',
```
Comment (jourdain) [inline webpack.config.js]: use env instead of es2015

File: webpack.config.js [hunk 3]
```diff
@@ -34,24 +42,48 @@ var loaders = [
...
   }, {
     test: /\.js$/,
     exclude: /node_modules/,
-    loader: 'babel?presets[]=es2015,presets[]=react',
+    loader: 'babel-loader?presets[]=es2015,presets[]=react',
```
Comment (jourdain) [inline webpack.config.js]: use env instead of es2015

File: webpack.config.js [hunk 4]
```diff
@@ -34,24 +42,48 @@ var loaders = [
...
   }, {
     test: /\.js$/,
     include: /node_modules(\/|\\)arctic-viewer(\/|\\)/,
-    loader: 'babel?presets[]=es2015,presets[]=react',
+    loader: 'babel-loader?presets[]=es2015,presets[]=react',
```
Comment (jourdain) [inline webpack.config.js]: use env instead of es2015

File: webpack.config.js [hunk 5]
```diff
@@ -34,24 +42,48 @@ var loaders = [
...
   }, {
     test: /\.js$/,
     include: /node_modules(\/|\\)vtk.js(\/|\\)/,
-    loader: 'babel?presets[]=es2015,presets[]=react',
+    loader: 'babel-loader?presets[]=es2015,presets[]=react',
```
Comment (jourdain) [inline webpack.config.js]: use env instead of es2015

**Comment by jourdain** [comment]
webpack 4 require a `--mode production/development` in the arg of the cli.

**Comment by cjh1** [comment]
It doesn't need to be required? ```webpack``` works?

**Review by cryos — APPROVED** [review]
FWIW I approve this PR :+1:

**Pull request merged**